vulcan/formula.py

```diff
@@ -75,14 +77,19 @@ def install(self, force=False):
                 shutil.rmtree(destination)
 
             # Move formula into place.
+            log.info('Moving formula %s to %s' %(self.name, destination))
             origin = os.path.join(tmpdir, self.origin_name)
             shutil.move(origin, destination)
 
             # Save state file.
             with open(self.state_file, 'w') as fh:
                 json.dump(self.as_dict(), fh)
 
+        except Exception as e:
```

Comment:
This is a dangerous change. If an exception is thrown between lines 71 and 86, this change will swallow the problem and just log an error message.

If this tool were running under something like Jenkins, it would log the error but produce a successful build because nothing is terminating the process with a value > 0.

I would probably just remove this but if you find value in the extra log message, it should [re-throw the exception](https://nedbatchelder.com/blog/200711/rethrowing_exceptions_in_python.html).

Generally, I don't catch exceptions unless I can really handle them like a re-try or using a different method for something e.g. (using curl instead of wget). If you don't know how to handle an exception, it's probably better to let it bubble all the way up to something that can or let it spew to stderr and terminate. When something totally unexpected happens, it's generally better to exit immediately with a code > 0.